jetty-core/jetty-openid/src/test/java/org/eclipse/jetty/security/openid/OpenIdProvider.java

```diff
@@ -1,445 +0,0 @@
-//
-// ========================================================================
-// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
-//
-// This program and the accompanying materials are made available under the
-// terms of the Eclipse Public License v. 2.0 which is available at
-// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
-// which is available at https://www.apache.org/licenses/LICENSE-2.0.
-//
-// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
-// ========================================================================
-//
-
-package org.eclipse.jetty.security.openid;
-
-import java.io.IOException;
-import java.time.Duration;
-import java.time.Instant;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.UUID;
-
-import org.eclipse.jetty.http.BadMessageException;
-import org.eclipse.jetty.http.HttpHeader;
-import org.eclipse.jetty.http.HttpStatus;
-import org.eclipse.jetty.server.Request;
-import org.eclipse.jetty.server.Response;
-import org.eclipse.jetty.server.Server;
-import org.eclipse.jetty.server.ServerConnector;
-import org.eclipse.jetty.server.handler.ContextHandler;
-import org.eclipse.jetty.server.handler.ContextHandlerCollection;
-import org.eclipse.jetty.util.BufferUtil;
-import org.eclipse.jetty.util.Callback;
-import org.eclipse.jetty.util.Fields;
-import org.eclipse.jetty.util.StringUtil;
-import org.eclipse.jetty.util.component.ContainerLifeCycle;
-import org.eclipse.jetty.util.statistic.CounterStatistic;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-public class OpenIdProvider extends ContainerLifeCycle
```

Comment:
I think we should leave the user facing classings in the old location as deprecated and extending the new classes.  This doesn't need to be all the classes, just ones that a user would have interacted with.

Reply:
We don't need to deprecate it because it's a test class in `src/test/java`.

Reply:
I was just picking this as the first class in the list... it is a general comment.  I see no deprecated classes, when I think there should be at least some.

Reply:
Do you want me to maintain `jetty-openid` under `jetty-core` and deprecate all the classes and have them extending the `jetty-integration` versions? If not I don't think any classes in this need to be deprecated.

Reply:
> Do you want me to maintain `jetty-openid` under `jetty-core` and deprecate all the classes and have them extending the `jetty-integration` versions?

Yes... or at least the ones that users might have used to configure it

Reply:
Actually, I can't even do that because there is no package change or maven coordinate change.

I don't think there is anything that needs to be deprecated.

Reply:
Oh yeah! Nothing is needed to be deprecated!